.github/workflows/dockerhub-release.yml

```diff
@@ -99,6 +99,7 @@ jobs:
             docker://${{ env.DH_IMAGE_REGISTRY }}/gitpod/$IMAGE:latest
           done
   notify:
+    if: always()
```

Comment:
this will also run and notify now even if `promote-latest`'s `if` is false (e.g. when dispatched from a branch), you could maybe check in `slack-failure` whether `promote-latest` was skipped, and then not notify?

Reply:
Good catch, thanks @WVerlaek ! I limited the `notify` job in 392a15d9b84dfc96cd252c47e8c1f3cb3faef80f to scheduled runs now (which use main), or workflow dispatch, where the context branch was main. This way, we'll still get notified on success, failure, and cancel.